doc/nrf/ug_thread_tools.rst

```diff
@@ -73,15 +73,22 @@ When working with samples that support wpantund, complete the following steps to
 
 1. Open a shell and run the wpantund process by using the following command:
 
-   .. parsed-literal::
-      :class: highlight
+   - connecting with Network Co-Processor real node located at ``/dev/ACM0`` which supports boudrate ``1000000``
 
-      wpantund -I *network_interface_name* -s *serial_port_name* -b *baudrate*
+      .. parsed-literal::
+         :class: highlight
+
+         sudo wpantund -I leader_if -s /dev/ACM0 -b 1000000
```

Comment:
is "sudo" really needed or are we only missing some extra configuration/permission settings?

Reply:
It's working without sudo but it reports permission warnings and log output is not complete.

Reply:
What more original documentation includes "sudo"